publish/publish.proj

```diff
@@ -241,6 +243,48 @@
                                  IgnoredErrorMessagesWithConditional="@(IgnorableErrorMessages)" />
   </Target>
 
+  <Target Name="DoPushCoreHostPackagesToAzure"
+          DependsOnTargets="DoPushPackagesToAzure;DoPushSymbolPackagesToAzure"
+          Condition="'$(OfficialPublish)' == 'true'"/>
+
+  <Target Name="DoPushPackagesToAzure">
+    <PropertyGroup>
+      <RelativePath>$(PackagesRelativePath)</RelativePath>
+      <RelativePath Condition="'$(RelativePath)' == ''">$(ConfigurationGroup)/pkg</RelativePath>
+      <ExpectedFeedUrl>https://$(TransportFeedAccountName).blob.core.windows.net/$(TransportFeedContainerName)/$(RelativePath)/index.json</ExpectedFeedUrl>
+      <PublishFlatContainer Condition="'$(PublishFlatContainer)' == ''">false</PublishFlatContainer>
+    </PropertyGroup>
+    <ItemGroup>
+      <ItemsToPush Remove="*.nupkg" />
+      <ItemsToPush Include="@(_DownloadedStandardPackages)"/>
+    </ItemGroup>
+    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
+                    AccountKey="$(TransportFeedAccessToken)"
+                    ItemsToPush="@(ItemsToPush)"
+                    PublishFlatContainer="$(PublishFlatContainer)"
+                    Overwrite="$(OverwriteOnPublish)" 
+                    IndexDirectory="$(ObjDir)"/>
+  </Target>
+
+  <Target Name="DoPushSymbolPackagesToAzure">
+    <PropertyGroup>
+      <RelativePath>$(SymbolsRelativePath)</RelativePath>
+      <RelativePath Condition="'$(RelativePath)' == ''">$(ConfigurationGroup)/symbolpkg</RelativePath>
+      <ExpectedFeedUrl>https://$(TransportFeedAccountName).blob.core.windows.net/$(TransportFeedContainerName)/$(RelativePath)/index.json</ExpectedFeedUrl>
```

Comment:
Did we figure out why this needs to be defaulted here? Isn't the build definition going to be passing it in?

Reply:
Otherwise we will not have the extensibility point we need to override the feed end-point